src/encord_active/lib/dataset/summary_utils.py

```diff
@@ -30,19 +30,62 @@ def get_median_value_of_2D_array(array: np.ndarray) -> np.ndarray:
+    labels, only counts the immediate labels.
     """
+    labels: dict = {"objects": {}, "classifications": {}}
     classification_label_counter = 0
     object_label_counter = 0
 
+    project_ontology = json.loads((project_folder / "ontology.json").read_text(encoding="utf-8"))
+    for object_item in project_ontology["objects"]:
+        labels["objects"][object_item["name"]] = 0
+    for classification_item in project_ontology["classifications"]:
+        labels["classifications"][classification_item["attributes"][0]["name"]] = {}
+
+        # For radio and checkbox types
+        for option in classification_item["attributes"][0].get("options", []):
+            labels["classifications"][classification_item["attributes"][0]["name"]][option["label"]] = 0
+
     for label_row in (project_folder / "data").iterdir():
         if (label_row / "label_row.json").exists():
             label_row_meta = json.loads((label_row / "label_row.json").read_text(encoding="utf-8"))
             if label_row_meta["data_type"] in [DataType.IMAGE.value, DataType.IMG_GROUP.value]:
                 for data_unit in label_row_meta["data_units"].values():
-                    object_label_counter += len(data_unit["labels"]["objects"])
-                    classification_label_counter += len(data_unit["labels"]["classifications"])
 
-    return classification_label_counter, object_label_counter
+                    object_label_counter += len(data_unit["labels"].get("objects", []))
+                    classification_label_counter += len(data_unit["labels"].get("classifications", []))
+
+                    for object_ in data_unit["labels"].get("objects", []):
+                        if object_["name"] not in labels["objects"]:
+                            print(f'Object name "{object_["name"]}" is not exist in project ontology')
+                        labels["objects"][object_["name"]] += 1
+
+                    for classification in data_unit["labels"].get("classifications", []):
+                        classificationHash = classification["classificationHash"]
+                        classification_answer_item = label_row_meta["classification_answers"][classificationHash][
+                            "classifications"
+                        ][0]
+                        classification_question_name = classification_answer_item["name"]
+                        if classification_question_name in labels["classifications"]:
+
+                            if isinstance(classification_answer_item["answers"], list):
+                                for answer_item in classification_answer_item["answers"]:
+                                    if answer_item["name"] in labels["classifications"][classification_question_name]:
+                                        labels["classifications"][classification_question_name][
+                                            answer_item["name"]
+                                        ] += 1
```

Comment:
if this is for handling checkboxes, this is probably not needed

Reply:
This is both for the radio and checklist types.